docs/mocksupport.rst

```diff
@@ -137,12 +137,13 @@ Pywbem mock does NOT support:
 1. CIM-XML protocol security and security constructor parameters of
    :class:`~pywbem.WBEMConnection`.
 2. Dynamic WBEM server providers in that the data for responses is from the
-   mock repository that is built before the request rather than from resources
-   accessed by such providers. This means there is no dynamic determination of
-   property values for newly created CIM instances. Therefore, creating CIM
-   instances requires that all keybinding values as well as all other property
-   values must be provided as input to the
-   :meth:`~pywbem.WBEMConnection.CreateInstance` operation by the user.
+   mock repository that is built before the request or internal to the
+   user-defined provider rather than from resources accessed by such providers.
+   This means there is no dynamic determination of property values for newly
+   created CIM instances. Therefore, creating CIM instances requires that all
+   keybinding values as well as all other property values must be provided as
+   input to the :meth:`~pywbem.WBEMConnection.CreateInstance` operation by the
+   user.
```

Comment:
If there are user-defined instance write providers, they can determine key properties, so this statement is not entirely true.

Reply:
Removed it when I removed that section.